Observatory-Control-System/BLOCKS/block-jira-project-workflow.rst

```diff
@@ -0,0 +1,245 @@
+For example, if a JSON BLOCK has been prepared to execute the test, this should be something like:
+* Run the JSON BLOCK using the add_block.py SAL Script with the following configuration - id: BLOCK-123
+
+If the test execution involves a series of SAL Scripts, provide the script names and all required configurations. 
+In cases where the user is expected to edit the configuration (e.g. to select a target), a full example configuration and possible list of configurables (e.g. target list) must be provided.
+
+Users are again encouraged to make use of the formatting options here. 
+All Script Configuration should be provided by inserting Code Snippets with YAML formatting. 
+
+**Description of Test Steps**
+This section should provide some context/expectations of what will happen during the test, including possible outcomes. 
+This will allow the users to evaluate the progress and success of the test without having to search and parse the detailed code. 
+It is  recommended to provide this as a bulleted list of steps, rather than in paragraph form, to make it easier to read. 
+
+BLOCK Ticket States and Transitions
+===================================
+
+The JIRA workflow for BLOCK Tickets has been designed to facilitate task preparation, planning, and execution. 
+BLOCK tickets provide the direct link between an observing task Proposer and the team that will be responsible for executing the task at the summit. 
+Here we describe the individual states and their allowed transitions. 
+In each section, we identify the work expected to be done in each state and the party responsible. 
+
+Active states
+-------------
+This section describes each of the active states as well as any transition from that state to another.
+
+**Proposed**
+^^^^^^^^^^^^
+This is the default state that a ticket enters when it is initially filed. 
+By default, the ticket is left unassigned. 
+The Commissioning Scientist team is responsible for the initial assessment and assignment of the observing task. 
+New ticket in the Proposed state will be reviewed and assigned on a weekly basis (or twice weekly) during one of several planning meetings.
+The reporter is not responsible for alerting anyone to new ticket creation, but a message on the #sitcom-observing-blocks slack channel is recommended. 
+Once the ticket has been assigned to a member of the Commissioning Science team, it will be transitioned to Preparation.
+  
+*Transition to Preparation*: An initial assessment of the observing task will be done during the weekly planning meeting by the Commissioning Science team. 
+This initial assessment will include a discussion of work needed to prepare the task, a decision on the priority of the task, and an assignment to a member of the Commissioning Science team for preparation.  
+Once the ticket has been assigned it can be transitioned to the Preparation state. 
+     
+
+**Preparation**
```

Comment:
Again, I thought we were going to remove this.

Reply:
Same comment, my notes say to keep this step.